### DISCUSSION TITLE
Anchor Component's showExternal Prop breaks word-wrap

### DISCUSSION THREAD

**Discussion by chjpark** [twilio-labs/paste#1533] [Q&A] [answered]
Hi,

I am using the Table component and am putting links in each table row. Occasionally the link names can get very long and needs to wrap the text so that it fits in the cell for the table row. I noticed that if I have the showExternal prop passed into the Anchor component, the word wrap breaks. I'm pretty sure this is a bug.

See the code sandbox:
https://codesandbox.io/s/this-is-a-test-4ccs5?file=/src/App.tsx

**Comment by richbachman** [comment]
Hi @chjpark. Apologies for the delay in getting back with you on this. Thanks for bringing this issue up. We'll take a look and get it fixed.

**Comment by richbachman** [comment, marked as answer]
Hi @chjpark. Thanks again for pointing out this issue. We're going to take a more in depth look at how we're composing the external icon. I'm creating a ticket in our backlog to figure it out.

In the meantime if you're blocked by this issue, feel free to remove the `showExternal` prop. You could also compose your own external anchor composition if its necessary. I've built a CodeSandbox showing how you could accomplish that: https://codesandbox.io/s/black-fog-ghpey?file=/src/index.tsx

**Comment by TheSisb** [comment]
Hi @chjpark,

Reporting back from the future :P  I just [merged a fix for our Anchors](https://github.com/twilio-labs/paste/pull/1969) so they wrap correctly as of the next release.
